### ISSUE TITLE
Training trend seems weird on COCO Datasets

### ISSUE THREAD

**Issue by daun-io** [tucan9389/tf2-mobile-pose-estimation#49]
This is my configuration file rewritten as the yaml format for my own convenience.

```yaml
dataset:
  dataset_root_path: /datasets/
  dataset_directory_name: coco_dataset
  train_images: train2017
  train_annotation: annotations_trainval2017/person_keypoints_train2017.json
  valid_images: val2017
  valid_annotation: annotations_trainval2017/person_keypoints_val2017.json

model:
  model_name: cpm
  model_subname: 
  batch_size: 3
  input_width: 192
  input_height: 192
  output_width: 24
  output_height: 24

preprocessing:
  is_scale: True
  is_rotate: True
  is_flipping: True
  is_resize_shortest_edge: True
  is_crop: True
  rotate_min_degree: -15.0
  rotate_max_degree: 15.0
  heatmap_std: 5.0

training:
  batch_size: 32
  learning_rate: 0.001
  epsilon: +1.e-8
  number_of_epoch: 200
  period_echo: 100
  period_save_model: 5000
  period_tensorboard: 10
  period_valid_image: 5000
  valid_pckh: True
  pckh_distance_ratio: 0.5
  multiprocessing_num: `4`
```

I've trained CPM model for days using this configuration and when I looked at the tensorboard the loss score and PCKh value showed no improvements over the course of training.

![image](https://user-images.githubusercontent.com/21135812/85963945-b057f500-b9f2-11ea-9649-0c5e02efc034.png)

Of course the heatmap result somehow showed the same results.
![result2-090000](https://user-images.githubusercontent.com/21135812/85963971-d1204a80-b9f2-11ea-85c6-a71a511c76f3.jpg)

When I changed my dataset from coco to ai_challenger dataset, the model seemed to be training fine.
![image](https://user-images.githubusercontent.com/21135812/85964001-f319cd00-b9f2-11ea-995d-3737cacc03b6.png)

I'm thinking that the error is caused from the data preparation steps however did not find specific codes to fix. Do you have an idea for this?

**Comment by daun-io** [comment]
I just figured out that the data loading modules are only compatible for ai challenger dataset yet.
